runtime/onert/backend/cpu/ops/MaxLayer.cc

```diff
@@ -38,12 +38,16 @@ void MaxLayer::maxFloat32()
 
 void MaxLayer::maxQuant8()
 {
-  // TODO Check whether cker for quant8 max produces correct results
-  // nnfw::cker::Max<uint8_t>(
-  //     getTensorShape(_lhs), reinterpret_cast<const uint8_t*>(_lhs->buffer()),
-  //     getTensorShape(_rhs), reinterpret_cast<const uint8_t*>(_rhs->buffer()),
-  //     getTensorShape(_output), reinterpret_cast<uint8_t*>(_output->buffer()));
-
+  if (_lhs->data_scale() == _rhs->data_scale() && _lhs->data_scale() == _output->data_scale())
+  {
+    if (_lhs->data_offset() == _rhs->data_offset() && _lhs->data_offset() == _output->data_offset())
+    {
+      return nnfw::cker::Max<uint8_t>(
+          getTensorShape(_lhs), reinterpret_cast<const uint8_t *>(_lhs->buffer()),
+          getTensorShape(_rhs), reinterpret_cast<const uint8_t *>(_rhs->buffer()),
+          getTensorShape(_output), reinterpret_cast<uint8_t *>(_output->buffer()));
+    }
+  }
   throw std::runtime_error("Max NYI for quantized");
```

Comment:
Please delete it.

Reply:
Probably, it is necessary because above `if` block doesn't support qaunt8 fully.

Reply:
I left this line for cases where the scale and offset are different.
I am not sure if I need to support some cases where the scale and offset are different.
Solving this will remove this line : )